app/res/xml/preferences_developer.xml

```diff
@@ -130,5 +130,10 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-bulk-performance"
         android:title="Enable cutting-edge performance improvements"/>
-
+    <ListPreference
```

Comment:
@ctsims The original implementation of this that got overriden had the `defaultValue` for this `ListPreference` set to `"yes"`. This seemed wrong to me, particularly since the `shouldHideReportIssue()` method was implemented to call `doesPropertyMatch()` with `NO` as the default value, so I've removed that in this version. But I just want to make sure I'm not missing something.